libmscore/barline.cpp

```diff
@@ -100,6 +100,10 @@ static void undoChangeBarLineType(BarLine* bl, BarLineType barType, bool allStav
             case BarLineType::END_REPEAT:
                   m->undoChangeProperty(Pid::REPEAT_END, true);
                   break;
+            case BarLineType::END_START_REPEAT:
+                  m->undoChangeProperty(Pid::REPEAT_END, true);
+                  m->undoChangeProperty(Pid::REPEAT_START, true);
```

Comment:
if (m->nextMeasure()) { m->nextMeasure()->undoChangeProperty(Pid::REPEAT_START, true); }

would be my first guess here

Reply:
Hmm, yes, that halfway solves the 2nd issue

Reply:
Hmm, I can't see how that wouldn't work completely.  This is, after all, exactly what happens if you add the two symbols by hand, and it looks fine then, does it not?  I don't see how it could be different if added this way in a single step versus if added in two separate steps.